csympy/lib/csympy_wrapper.pyx.in

```diff
@@ -6,6 +6,12 @@ from libcpp.string cimport string
 from cpython cimport PyObject, Py_XINCREF, Py_XDECREF, \
     PyObject_CallMethodObjArgs
 
+DEF HAVE_SAGE = ${HAVE_SAGE}
+have_sage = ${HAVE_SAGE}
```

Comment:
Yes, I would like to actually shift to use `setup.py`, just for the extension. But that's a separate issue (#413).